src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamTests.cs

```diff
@@ -617,6 +617,12 @@ public async Task WritePreCanceled_Throws()
         [Fact]
         public async Task WriteCanceled_NextWriteThrows()
         {
+            // [ActiveIssue("https://github.com/dotnet/runtime/issues/55995")]
+            if (typeof(T) == typeof(MsQuicProviderFactory))
```

Comment:
Should it be Mock, not MsQuic?

Reply:
Of course it should! Thanks for catching that!